lib/workers/branch/execute-post-upgrade-commands.ts

```diff
@@ -37,11 +38,16 @@ export async function postUpgradeCommandsExecutor(
     );
     const commands = upgrade.postUpgradeTasks?.commands || [];
     const fileFilters = upgrade.postUpgradeTasks?.fileFilters || [];
-
     if (is.nonEmptyArray(commands)) {
       // Persist updated files in file system so any executed commands can see them
       for (const file of config.updatedPackageFiles.concat(updatedArtifacts)) {
-        if (file.name !== '|delete|') {
+        let stat;
```

Comment:
Needs type

Reply:
This place in code doesn't exist anymore